receiver/k8sclusterreceiver/internal/collection/collector.go

```diff
@@ -104,6 +107,14 @@ func (dc *DataCollector) CollectMetricData(currentTime time.Time) pmetric.Metric
 	dc.metadataStore.ForEach(gvk.ClusterResourceQuota, func(o any) {
 		clusterresourcequota.RecordMetrics(dc.metricsBuilder, o.(*quotav1.ClusterResourceQuota), ts)
 	})
+	dc.metadataStore.ForEach(gvk.HierarchicalResourceQuota, func(o any) {
+		hrq, err := hierarchicalresourcequota.Parse(o.(*unstructured.Unstructured))
```

Comment:
Please remind me: do we have to deal with `*unstructured.Unstructured` because we can't create a watcher that can return `*v1alpha2.HierarchicalResourceQuota`?

Reply:
Yes. There is no informer for v1alpha2.HierachicalResourceQuota in the [hierarchical-namespaces project](https://github.com/kubernetes-sigs/hierarchical-namespaces)

Reply:
And I see that youve added https://github.com/kubernetes-sigs/hierarchical-namespaces/issues/350, thank you!